Makefile

```diff
@@ -290,7 +290,7 @@ internal-gendependabot:
 	@echo "      interval: \"weekly\"" >> ${DEPENDABOT_PATH};
 
 .PHONY: gendependabot
-gendependabot:
+gendependabot: $(eval SHELL:=/bin/bash)
```

Comment:
This probably deserves a comment explaining why it is necessary so that the next person to work on this doesn't think it useless and remove it.